File: android/src/i/am/rauan/satanbek/spider/AndroidLauncher.java
package i.am.rauan.satanbek.spider;

import android.os.Bundle;

import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
import i.am.rauan.satanbek.spider.SpiderGame;

public class AndroidLauncher extends AndroidApplication {
	@Override
	protected void onCreate (Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		AndroidApplicationConfiguration config = new AndroidApplicationConfiguration();
		initialize(new SpiderGame(), config);
	}
}

File: android/src/i/am/rauan/satanbek/spider/AndroidLauncher.kt
package i.am.rauan.satanbek.spider

import android.os.Bundle

import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration
import i.am.rauan.satanbek.spider.SpiderGame

class AndroidLauncher : AndroidApplication() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val config = AndroidApplicationConfiguration()
        initialize(SpiderGame(), config)
    }
}
